lib/improver/tests/blending/weighted_blend/test_PercentileBlendingAggregator.py

```diff
@@ -42,46 +42,71 @@
 from improver.blending.weighted_blend import PercentileBlendingAggregator
 
 PERCENTILE_DATA = np.array([
-    15.3077946, 14.65380361, 15.91478244, 15.10887522,
-    12.30817311, 14.61048935, 17.80547685, 15.16351283,
-    17.65229058, 18.16963972, 19.96189077, 19.87125985,
-    18.14138889, 16.66496889, 19.12200888, 18.97425367,
-    18.88194635, 18.63268387, 21.23771814, 21.22492499,
-    18.64451978, 18.08331902, 20.0182035, 19.95292475,
-    21.4430201, 19.89146966, 22.11073569, 22.84817844,
-    19.50214911, 19.63833851, 21.1260302, 21.19184589,
-    22.64466784, 21.18604047, 22.55922723, 24.00842263,
-    21.11200487, 20.99792316, 22.07955526, 21.71937432,
-    23.12910858, 22.94415916, 25.11682828, 25.76481447,
-    23.82016292, 22.11742526, 26.30864878, 23.81492893])
-
-BLENDED_PERCENTILE_DATA1 = np.array([
-    12.308173106352402, 14.610489348891615,
-    15.914782442039872, 15.108875220912573,
-    17.737443276652936, 17.097492097934627,
-    19.274275011968893, 19.127165271387128,
-    18.742074248084133, 18.50663225732617,
-    20.584984938812934, 20.560776453138736,
-    20.25126805925584, 19.769776454949138,
-    21.64008410795689, 21.598099315407907,
-    22.17361605670607, 21.085160674433627,
-    22.51323580222509, 23.192691941719808,
-    23.82016292415821, 22.944159163666185,
-    26.30864878110861, 25.76481446912205])
-
-BLENDED_PERCENTILE_DATA2 = np.array([
-    12.30817311, 14.61048935,
-    15.91478244, 15.10887522,
-    17.676775957972925, 17.593947064474417,
-    19.51651034470578, 19.378940557706372,
-    18.819297828987274, 18.594620794715546,
-    20.965096170815123, 20.95201851550872,
-    20.890749057567294, 19.843891344879836,
-    21.927325497880343, 22.32506953618696,
-    22.491628272758373, 21.143852884603188,
-    22.546838489123576, 23.730102965015696,
-    23.82016292, 22.94415916,
-    26.30864878, 25.76481447])
+    17.458706, 13.732982, 15.138694, 13.966815, 16.187801,
```

Comment:
It might be worth adding a few comments about where these numbers have come from i.e. how they've been created, in case we need to recreate them later.